client/src/store.js

```diff
@@ -2,11 +2,14 @@ import { createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
 import rootReducer from './reducers/rootReducer';
 
-// Defined an initial state just as an example and to test functionality
-export default function configureStore(initialState = { placeholder: 'redux' }) {
+// Allows dev to see state in real time in redux dev tools
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Where is `compose` defined? I'm seeing this console error: `Uncaught ReferenceError: compose is not defined`

Reply:
Whoops! Forgot to define it above. The first line should look like this: 
 `import { createStore, applyMiddleware, compose } from 'redux';`